src/sentry/static/sentry/app/data/platformCategories.tsx

```diff
@@ -93,14 +103,17 @@ export const sourceMaps: PlatformKey[] = [
   'electron',
 ];
 
-export const tracing = ['python-tracing', 'node-tracing'] as const;
+// TODO(epurkhiser): I've added these since there was a need to have them in
+// the platform key type. However I have not added them anywhere else.
```

Comment:
Did I write this comment? Where did it come from 🤔

Reply:
lol